backend/app/services/pet/ws_emitter.py

```
@@ -8,14 +8,55 @@
+
+    main_loop = get_main_event_loop()
+    if not main_loop:
+        logger.warning("[PET_WS] Main event loop not available, skipping emit")
+        return
+
+    try:
+        # Check if we're already in the main loop
+        current_loop = asyncio.get_running_loop()
+        if current_loop is main_loop:
+            # Same loop - just create task
+            asyncio.create_task(coro)
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)